controller/src/main.cpp

```diff
@@ -60,6 +61,31 @@ static ControllerStatus controller_status = ControllerStatus_init_zero;
 // Last-received status from the GUI.
 static GuiStatus gui_status = GuiStatus_init_zero;
 
+// desired actuators state
+static ActuatorsState actuators_state;
+
+class Controller {
+public:
+  ActuatorsState Run(Time now, VentParams params, SensorReadings readings) {
+    // TODO: Add additional fields to the ControllerStatus proto:
+    //   - Blower speed (0-255 PWM value)
+    //   - Expiratory solenoid valve state (open/closed).
+    BlowerSystemState desired_state = blower_fsm_desired_state(now, params);
+
+    // pass setpoint to controller status for testing reasons
+    controller_status.fan_setpoint_cm_h2o =
+        desired_state.setpoint_pressure.cmH2O();
+
+    // output struct
```

Comment:
comment isn't necessary

Reply:
updated return with the same trick as the one in get_sensor_readings and of course removed the comment